model/host/hostutil.go

```diff
@@ -566,9 +576,34 @@ func (h *Host) WriteJasperCredentialsFilesCommands(splunk send.SplunkConnectionI
 	return strings.Join(cmds, " && "), nil
 }
 
-// WriteJasperCredentialsFilesCommands is the same as
+func bufferedWriteFileCommands(path, content string) []string {
```

Comment:
Is this covered by an existing unit test?

Reply:
I added a test, but it's not a very useful one.